python/paddle/incubate/layers/__init__.py

```diff
@@ -0,0 +1,53 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import nn
+from .nn import (
+    fused_embedding_seq_pool,
+    fused_seqpool_cvm,
+    multiclass_nms2,
+    search_pyramid_hash,
+    shuffle_batch,
+    partial_concat,
+    partial_sum,
+    tdm_child,
+    tdm_sampler,
+    rank_attention,
+    batch_fc,
+    _pull_box_extended_sparse,
+    bilateral_slice,
+    correlation,
+    fused_bn_add_act,
+    pow2_decay_with_linear_warmup,
+    _pull_gpups_sparse,
+    _pull_box_sparse,
+)
+
+__all__ = [
+    'fused_embedding_seq_pool',
+    'multiclass_nms2',
+    'search_pyramid_hash',
+    'shuffle_batch',
+    'partial_concat',
+    'partial_sum',
+    'tdm_child',
+    'rank_attention',
+    'tdm_sampler',
+    'batch_fc',
+    'bilateral_slice',
+    'correlation',
+    'fused_bn_add_act',
+    'fused_seqpool_cvm',
+    'pow2_decay_with_linear_warmup',
+]
```

Comment:
Should these all be public APIs? If yes, Chinese documents need to be written and shown in official website.

Reply:
After offline discussion, these APIs are not treated as public and will be maintained by the corresponding group. `__all__` list are already cleared.